tests/nlu/test_config.py

```diff
@@ -99,43 +101,78 @@ def test_default_config_file():
     assert len(final_config) > 1
 
 
-def test_set_attr_on_component(pretrained_embeddings_spacy_config):
-    idx_classifier = pretrained_embeddings_spacy_config.component_names.index(
-        "SklearnIntentClassifier"
-    )
-    idx_tokenizer = pretrained_embeddings_spacy_config.component_names.index(
-        "SpacyTokenizer"
+def test_set_attr_on_component():
+    _config = RasaNLUModelConfig(
+        {
+            "language": "en",
+            "pipeline": [
+                {"name": "SpacyNLP"},
+                {"name": "SpacyTokenizer"},
+                {"name": "SpacyFeaturizer"},
+                {"name": "RegexFeaturizer"},
+                {"name": "CRFEntityExtractor"},
+                {"name": "EntitySynonymMapper"},
+                {"name": "SklearnIntentClassifier"},
+            ],
+        }
     )
-    pretrained_embeddings_spacy_config.set_component_attr(idx_classifier, C=324)
+    idx_classifier = _config.component_names.index("SklearnIntentClassifier")
+    idx_tokenizer = _config.component_names.index("SpacyTokenizer")
 
-    assert pretrained_embeddings_spacy_config.for_component(idx_tokenizer) == {
-        "name": "SpacyTokenizer"
-    }
-    assert pretrained_embeddings_spacy_config.for_component(idx_classifier) == {
+    _config.set_component_attr(idx_classifier, C=324)
+
+    assert _config.for_component(idx_tokenizer) == {"name": "SpacyTokenizer"}
+    assert _config.for_component(idx_classifier) == {
         "name": "SklearnIntentClassifier",
         "C": 324,
     }
 
 
-def test_override_defaults_supervised_embeddings_pipeline(supervised_embeddings_config):
+def test_override_defaults_supervised_embeddings_pipeline():
```

Comment:
should we test override on different component? since we plan to deprecate sklearn